### ISSUE TITLE
override keyword gets removed for members in constructor

### ISSUE THREAD

**Issue by Flarna** [prettier/prettier#11017]
**Prettier 2.3.1**
[Playground link](https://prettier.io/playground/#N4Igxg9gdgLgprEAucAbAhgZ0wAgEI7AA6UOOkUmMATgK5gwTUAUADtRPA3ACY7Vx0PaKgCe5aADMAlgHMkhHOgVVq0qLJwBfAJSEtJA1BJgM2HABEccAB7woPXAWKkJlGvUYt2nONz4QAG5w1Go8cPyCwlBibjLyiso4quqyANw4AEYKULQAtpkh2nouZGSYtKwhzBTxOmkkZEZGIAA0IBCsMNLQmMig6KEQAO4ACoMIfSjoqMPoon3tmdToYADWcDAAyqyrqcgecO1wBbzhPAAy6Bq06LJwAGJMeegw3RrIIOi0jG0gABYwPKoADq-2k8EwuzAcC2kwh0kCENEnzA2D+6kwIRgoxWsheyEkMyx7QAVpgbHgVutNlt0Hk4Bd1HBCcSjiByTYtqlUHAAIq0Xys1AkkC7ahY6ifGCiKqYMBqLp-djqGAg6Q8GD-ZAADgADO0fFiQStWJ92HBJcE-gBHQXwXGdKZfTAAWigcF4vD+Ajt0gEuLuBKQRJF7KxeWkBzo4Z5-PtLJDbPaMHQmXVmu1SAATCmVtJUKkAMIQPLBkCWgCsf1oWIAKmmpqHRYFaHAAJIOBDbBXSLoAQQcWxlvOFWK0WiAA)

```sh
--parser typescript
```

**Input:**

```tsx
class B {
  constructor(protected readonly config: { a: string }) {}
}

class D extends B {
  constructor(protected override readonly config: { a: string; b: number }) {
    super(config);
  }
}

```

**Output:**

```tsx
class B {
  constructor(protected readonly config: { a: string }) {}
}

class D extends B {
  constructor(protected readonly config: { a: string; b: number }) {
    super(config);
  }
}

```

**Expected behavior:**
`override` keyword should be preserved.

**Comment by sosukesuzuki** [comment]
I've created Pull Requests to two TypeScript parsers we depends:

- https://github.com/typescript-eslint/typescript-eslint/pull/3485
- https://github.com/babel/babel/pull/13428

**Comment by Flarna** [comment]
Thanks a lot for the fix!
May I ask if there is a new release planed which includes this?

**Comment by sosukesuzuki** [comment]
We did not plan for this, but I think this is an important bug and we should release it :+1:

**Comment by sosukesuzuki** [comment]
@Flarna Sorry for delay, I've published Prettier 2.3.2 that supports override modifier for parameter properties!
